src/module/Ui/templates/authentication/login.twig

```diff
@@ -14,13 +14,13 @@
 <div class="form-group">
     <div class="input-group">
         <span class="input-group-addon"><i class="fa fa-user"></i></span>
-        {{ formElement(form.get('email')) }}
+        {{ formElement(form.get('email').setAttribute('placeholder', 'Email'|trans)) }}
```

Comment:
Ah yes I know that problem, there is a helper function for that: https://github.com/serlo-org/athene2/blob/master/src/public/index.php#L34

So just do `setAttribute('a', t('b'))`